example/dummy/dummy.go

```diff
@@ -32,6 +34,7 @@ func (app *DummyApplication) DeliverTx(tx []byte) types.Result {
 	} else {
 		app.state.Set(tx, tx)
 	}
+	fmt.Println("set data")
```

Comment:
did you use it for debug or we need this?

Reply:
Debug. Oops